src/listEditing.ts

```diff
@@ -21,6 +23,19 @@ function isInFencedCodeBlock(doc: TextDocument, lineNum: number): boolean {
     }
 }
 
+async function getLengthChange(doc: TextDocument, lineNum: number) {
+    let beforeLen = doc.lineAt(lineNum).text.length;
+    await commands.executeCommand('cursorMove', {to: 'left', value: 1});
+    await commands.executeCommand('cursorMove', {to: 'right', value: 1});
+    return (doc.lineAt(lineNum).text.length - beforeLen);
+}
+
+async function handleUnikey(editor: vscode.TextEditor, range: Range) {
+    return await editor.edit(editBuilder => {
+        editBuilder.delete(range);
+    });
+}
+
```

Comment:
Could you please explain what these two functions are doing?

(Move my comments to this line)

Reply:
getLengthChange -> When typing Vietnamese, it will insert Vietnamese characters. I need to check how many characters inserted.
handleUnikey -> wait for the editor call delete successful.